docs/src/installation.md

```diff
@@ -2,47 +2,73 @@
 
 If using Martin with PostgreSQL database, you must install PostGIS with at least v3.0+, v3.1+ recommended.
 
-### Binary Distributions
+### Docker
+
+Martin is also available as a [Docker image](https://ghcr.io/maplibre/martin). You could either share a configuration file from the host with the container via the `-v` param, or you can let Martin auto-discover all sources e.g. by passing `DATABASE_URL` or specifying the .mbtiles/.pmtiles files or URLs to .pmtiles.
+
+```shell
+export PGPASSWORD=postgres  # secret!
+docker run -p 3000:3000 \
+           -e PGPASSWORD \
+           -e DATABASE_URL=postgresql://user@host:port/db \
+           -v /path/to/config/dir:/config \
+           ghcr.io/maplibre/martin --config /config/config.yaml
+```
+
+### From Binary Distributions Manually
 
 You can download martin from [GitHub releases page](https://github.com/maplibre/martin/releases).
 
-| Platform | Downloads (latest)     |
-|----------|------------------------|
-| Linux    | [64-bit][rl-linux-tar] |
-| macOS    | [64-bit][rl-macos-tar] |
-| Windows  | [64-bit][rl-win64-zip] |
+| Platform | AMD-64                                                                                           | ARM-64                              |
```

Comment:
AMD-64 is a bit confusing (although technically correct). Perhaps use the `x64` is better, vs `arm-64`?